docs/metadata.md

```diff
@@ -0,0 +1,161 @@
+```
+strain              virus  gisaid_epi_isl  genbank_accession   date        region   ...
+NewZealand/01/2020  ncov   EPI_ISL_413490  ?                   2020-02-27  Oceania  ...
+```
+
+In total there are 23 columns of metadata for each genome.
+Let's jump in and explore them, using the above example 👇
+
+**Column 1: `strain`**
+
+This needs to match the name of a sequence in the FASTA file exactly and must not contain characters such as spaces, or `()[]{}|#><`.
+In our example we have a strain called "NewZealand/01/2020" so there should be a sequence in the FASTA file for ">NewZealand/01/2020" (sequence names in FASTA files always start with the `>` character, but this is not part of the name).
+
+**Note that "strain" here carries no biological or functional significance** and should be thought of as synonymous with sample.
+
+**Column 2: `virus`**
+
+Currently unused.
+
+**Column 3: `gisaid_epi_isl`**
+
+If this genome is shared via [GISAID](https://www.gisaid.org/) then please include the EPI ISL here. In our example this is "EPI_ISL_413490".
+
+**Column 4: `genbank_accession`**
+
+If this genome is shared via [GenBank](https://www.ncbi.nlm.nih.gov/genbank/) then please include the accession number here. In our example this is "?" indicating that it hasn't (yet) been deposited in GenBank. (See above for more information on how to encode missing data.)
+
+**Column 5: `date`** (really important!)
+
+This describes the sample collection data (_not_ sequencing date!) and must be formated according as `YYYY-MM-DD`. 
+Our example was collected on Feb 27, 2020 and is therefore represented as "2020-02-27".
+
+You can specify unknown dates or month by replacing the respected values by `XX` (ex: `2013-01-XX` or `2011-XX-XX`) and completely unknown dates can be shown with `20XX-XX-XX` (which does not restrict the sequence to being in the 21st century - they could be earlier).
+Please be aware that our current pipeline will filter out any genomes with an unknown date, however you can change this for your pipeline!
+
+**Column 6: `region`**
+
+The region the sample was collected in -- for our example this is "Oceania".
+Please use one of "Africa", "Asia", "Europe", "North America", "Oceania" or "South America".
+If you sequence a genome from Antartica please get in touch!
+
+
+**Column 7: `country`**
+
+The country the sample was collected in. Our example, "NewZealand/01/2020", was collected in ....... New Zealand.
+You can run `tail +2 data/metadata.tsv | cut -f 7 | sort | uniq` to see all the countries currently present in the metadata.
+As of April 10 there were 64! 🌎
+
+**Column 8: `division`**
+
+Division currently doesn't have a precise definition and we use it differently for different regions.
+For instance, for samples in the USA we are detailing the state in which the sample was collected here.
```

Comment:
```suggestion
For instance, for samples in the USA we are detailing the state in which the sample was collected here. For other countries, it might be a county, region, or other administrative sub-division.
```